index.html

```diff
@@ -353,40 +309,12 @@ <h3>
             total: {
```

Comment:
Maybe this example can be made more clear by adding a new display item for shipping? e.g.
```
displayItems: [
  { // sub-total },
  { // tax },
  {
    label: "Standard shipping",
    amount: { currency: "GBP", value: "5.00" },
  },
]
```